packages/arb-token-bridge-ui/src/components/TransactionHistory/EmptyTransactionHistory.tsx

```diff
@@ -0,0 +1,61 @@
+import { GET_HELP_LINK } from '../../constants'
+import { ExternalLink } from '../common/ExternalLink'
+import {
+  ContentWrapper,
+  HistoryLoader,
+  LoadMoreButton
+} from './TransactionHistoryTable'
+
+export const EmptyTransactionHistory = ({
+  loading,
+  isError,
+  paused,
+  resume,
+  tab
+}: {
+  loading: boolean
+  isError: boolean
+  paused: boolean
+  resume: () => void
+  tab: 'pending' | 'settled'
+}) => {
+  if (loading) {
+    return (
+      <ContentWrapper>
+        <HistoryLoader />
+      </ContentWrapper>
+    )
+  }
+  if (isError) {
+    return (
+      <ContentWrapper>
+        <p>
+          We seem to be having a difficult time loading your data, we&apos;re
+          working hard to resolve it.
+        </p>
+        <p>Please give it a moment and then try refreshing the page.</p>
+        <p className="mt-4">
+          If the problem persists, please file a ticket{' '}
+          <ExternalLink className="arb-hover underline" href={GET_HELP_LINK}>
+            here
+          </ExternalLink>
+          .
+        </p>
+      </ContentWrapper>
+    )
+  }
+  if (paused) {
+    const isPendingTab = tab === 'pending'
+
+    return (
+      <ContentWrapper className="space-y-4">
+        <p>
+          There are no recent {isPendingTab ? 'pending' : 'settled'}{' '}
+          transactions.
+        </p>
+        <LoadMoreButton onClick={resume} />
+      </ContentWrapper>
+    )
+  }
+  return <ContentWrapper>Looks like no transactions here yet.</ContentWrapper>
```

Comment:
NIT: The grammar seems off. But we can do a complete language audit in the main branch, hence not a blocker.

Reply:
Yeah it does seem off to me too. I even asked chatgpt but it said the grammar is fine to me. Still not convinced, I will ask the others to confirm

Reply:
grammatically it's correct i think because it's an imperative sentence

but we can change it to "Looks like there are no transactions yet."